ChangeLog

```diff
@@ -13,6 +13,16 @@
 
 2015-06-14  Titus Brown  <titus@idyll.org>
 
+   * scripts/extract-paired-reads.py: added --output_dir, --paired-output,
```

Comment:
I tried doing a coverage check on my laptop but it said all changed  lines
were uncovered which seems unlikely.

On Fri, Jun 26, 2015, 17:30 C. Titus Brown <notifications@github.com> wrote:

> In ChangeLog
> <https://github.com/dib-lab/khmer/pull/1085#discussion_r33406746>:
>
> > @@ -13,6 +13,16 @@
> >
> >  2015-06-14  Titus Brown  <titus@idyll.org>
> >
> > +   * scripts/extract-paired-reads.py: added --output_dir, --paired-output,
>
> ok shall I make this mergable/fix ChangeLog, or are there other things
> that need to be done?
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/dib-lab/khmer/pull/1085/files#r33406746>.
>
-- 
Michael R. Crusoe: Programmer & Bioinformatician crusoe@ucdavis.edu
The lab for Data Intensive Biology; University of California, Davis
https://impactstory.org/MichaelRCrusoe http://twitter.com/biocrusoe